aidants_connect_web/models.py

```diff
@@ -1270,8 +1270,9 @@ def save(
                         f"french region among {settings.FRENCH_REGION_CODES}"
                     )
 
-            # Normalize phone number to international format
             self.user_phone = format_number(user_phone, PhoneNumberFormat.E164)
+            if update_fields is not None:
+                update_fields = {"user_phone"}.union(update_fields)
```

Comment:
https://docs.djangoproject.com/en/4.2/releases/4.2/#setting-update-fields-in-model-save-may-now-be-required